Repositories/MSSQLProductRepository.cs

```diff
@@ -0,0 +1,123 @@
+﻿using IMS.Repositories;
+using Microsoft.Data.SqlClient;
+using System.Data;
+namespace IMS
+{
+    public class MSSQLProductRepository : IProductRepository
+    {
+        private string connectionString = "Server=DESKTOP-CUQN3UP\\SQLEXPRESS;Database=IMS;Integrated Security=True;TrustServerCertificate=true;";
```

Comment:
let's create `InventoryDbContext` that inherits `DbContext` and have this in the `OnConfiguring` override